src/main/@types/slack.d.ts

```diff
@@ -0,0 +1,2 @@
+type PostType = { status_text: string; status_emoji: string; status_expiration: number };
```

Comment:
1) Can this instead be `export type ...` to achieve both the type Creation + sharing it.
2) is it possible to have some new lines in this, just for readability 😄 
3) Can this be renamed to **SlackPostData** or something.
   1) Prefix it with the Integration name (to not confuse it in Intelisense)
   2) `Type` shouldn't be needed to be included as that is what it's defined as.
   3) PostData is actually what this is being used for.